builtin/settingtypes.txt

```diff
@@ -2111,7 +2111,7 @@ curl_parallel_limit (cURL parallel limit) int 8 1 2147483647
 #    Maximum time a file download (e.g. a mod download) may take, stated in milliseconds.
 curl_file_download_timeout (cURL file download timeout) int 300000 5000 2147483647
 
-[**Misc]
+[**Misc.]
```

Comment:
name it "Miscellaneous"?
or are we short on space?

Reply:
The space should be enough for "Miscellaneous". It is located inside the main area (not inside the navigation panel).